plenum/common/messages/client_request.py

```diff
@@ -98,6 +99,14 @@ class ClientGetTxnAuthorAgreementOperation(MessageValidator):
     )
 
 
+class ClientGetTxnAuthorAgreementAMLOperation(MessageValidator):
```

Comment:
Please add a test for input validation of this (like in `test_client_taa_op.py`)

Reply:
ok